org.beeref.BeeRef.json

```diff
@@ -0,0 +1,63 @@
+{
+    "id": "org.beeref.BeeRef",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "beeref",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--env=PATH=/usr/bin:/app/bin"
+    ],
+    "build-options": {
+        "env": {
+            "PYTHON": "/usr/bin/python3"
+        },
+        "build-args": [
+          "--share=network"
+        ]
+    },
+    "modules": [
+        {
+            "name": "BeeRef",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir pipdir",
+                "${PYTHON} -m pip install -U pip",
+                "${PYTHON} -m pip install pyinstaller",
+                "${PYTHON} -m pip install -e .",
+                "${PYTHON} -m PyInstaller BeeRef.spec",
+
+                "install -D dist/BeeRef* /app/bin/beeref",
+                "install -Dm644 org.beeref.BeeRef.svg /app/share/icons/hicolor/scalable/apps/org.beeref.BeeRef.svg",
+                "install -Dm644 org.beeref.BeeRef.desktop /app/share/applications/org.beeref.BeeRef.desktop",
+                "install -Dm644 org.beeref.BeeRef.appdata.xml /app/share/metainfo/org.beeref.BeeRef.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/rbreu/beeref.git",
+                    "tag": "v0.2.0",
+                    "commit": "c308c08a07a6bce0ee1ece487f924811f1e56053"
+                },
+                {
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/rbreu/beeref/main/beeref/assets/logo.svg",
```

Comment:
this is already part of the git checkout above.

Reply:
Good point